# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/gcc/hotfix-no-multilib.patch.superh
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- gcc-11.2.0/gcc/config/sh/t-sh.vanilla	2021-12-30 22:55:30.742405660 +0100
+++ gcc-11.2.0/gcc/config/sh/t-sh	2021-12-30 23:03:11.775387277 +0100
@@ -39,6 +39,7 @@
 MULTILIB_OPTIONS= $(OTHER_ENDIAN) $(TM_MULTILIB_CONFIG)
 MULTILIB_DIRNAMES= 
 
+ifdef CRAZY_MULTILIB_CONFIG
 # The separate entries for m2a-nofpu and m2a-single-only with
 # duplicate base libraries are there to make sure we don't ever use an
 # m4* multilib for m2a or vice-versa; they are not compatible.  This
@@ -89,6 +90,8 @@
 	m4a=!m4a $(OTHER_ENDIAN)/m4a=!$(OTHER_ENDIAN)/m4a \
 	m4al=!m4al $(OTHER_ENDIAN)/m4al=!$(OTHER_ENDIAN)/m4al
 
+endif
+
 $(out_object_file): gt-sh.h
 gt-sh.h : s-gtype ; @true
 
